JS-1061_Making_Decisions_in_Your_Code_with_Conditional_Statements_challenge_2_pt3/scripts.js

```diff
@@ -0,0 +1,11 @@
+const isAdmin = true;
```

Comment:
This should be set to `false` in order for the conditional statement to fall into `else`.